2/ДЗ. TypeScript. Примітивні літеральні типи та типізація обʼєктів.ts

```diff
@@ -1,93 +1,225 @@
+interface Lecturer {
+  id: string | number
+  name: string
+  surname: string
+  position: string
+  company: string
+  experience: number
+  courses: string[]
+  contacts: string
+}
+
 class School {
   // implement 'add area', 'remove area', 'add lecturer', and 'remove lecturer' methods
 
-  _areas = [];
-  _lecturers = []; // Name, surname, position, company, experience, courses, contacts
+  private _areas: Area[] = []
+  private _lecturers: Lecturer[] = [] // Name, surname, position, company, experience, courses, contacts
+
+  get areas(): Area[] {
+    return this._areas
+  }
+
+  get lecturers(): Lecturer[] {
+    return this._lecturers
+  }
+
+  addArea(area: Area) {
+    this._areas.push(area)
+  }
 
-  get areas() {
-    this._areas;
+  removeArea(areaName: string) {
+    this._areas = this._areas.filter(area => area.getName() !== areaName)
   }
 
-  get lecturers() {
-    this._lecturers;
+  addLecturer(lecturer: Lecturer) {
+    this._lecturers.push(lecturer)
+  }
+
+  removeLecturer(lecturerId: string | number) {
+    this._lecturers = this._lecturers.filter(lecturer => lecturer.id !== lecturerId)
   }
 }
 
 class Area {
   // implement getters for fields and 'add/remove level' methods
-  _levels = [];
-  _name;
+  private _levels: Level[] = []
+  private _name: string
+
+  constructor(name: string) {
+    this._name = name
+  }
+
+  getName() {

```

Comment:
It's not a getter accessor. It should look like:
`get name(): Type { // your code }`